exchanges/bittrex/bittrex_wrapper.go

```diff
@@ -1010,6 +1009,13 @@ func (b *Bittrex) GetHistoricCandles(ctx context.Context, pair currency.Pair, a
 		ohlcData = append(ohlcData, historicData...)
 	}
 	if getRecent {
+		// This is a workaround so we don't get candle padding between
```

Comment:
I think it would be good to get a test on `TestGetHistoricCandles` to cover this

Also, is the purpose of this to see if you get an error because of the `true`?

Reply:
Will do, I have added this because getting recent you are restricted by the time window between the start value to time.Now(). This time window might be quite large and if its not checked, we might get a bunch of data cut off due to its max return. So we might get into a situation where data returned historically will be seen then a big gap with padding and then the recent data.